libqtile/widget/base.py

```diff
@@ -577,6 +585,14 @@ def _configure(self, qtile, bar):
         _Widget._configure(self, qtile, bar)
         if self.fontsize is None:
             self.fontsize = self.bar.height - self.bar.height / 5
+        if self.direction != "default" and self.direction != "ttb" and self.direction != "btt":
+            logger.warning(
+                """Invalid value set for direction: %s
+                               Valid values are: default, ttb, btt
+                               direction set to default""",
+                self.direction,
+            )
```

Comment:
```suggestion
            logger.warning(
                "Invalid value set for direction: %s. Valid values are: 'default', 'ttb', 'btt'. "
                "direction has been set to 'default'",
                self.direction,
            )
```
Don't think we need the log message to span over 3 lines.